Source/DafnyCore/TestGenerationOptions.cs

```diff
@@ -8,7 +8,7 @@ public class TestGenerationOptions {
     public const string TestInlineAttribute = "testInline";
     public const string TestEntryAttribute = "testEntry";
     public bool WarnDeadCode = false;
-    public enum Modes { None, Block, Path };
+    public enum Modes { None, Block, Branch, Path };
```

Comment:
You could call it `CallGraph` coverage?

Reply:
I think CallGraph works great, thank you! Have renamed it